MBBSEmu.Tests/ExportedModules/Majorbbs/stzcat_Tests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MBBSEmu.Memory;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class stzcat_Tests : ExportedModuleTestBase
+    {
+        private const int STZCAT_ORDINAL = 905;
+
+        [Theory]
+        [InlineData("12345", "12345", 7, "123451")]
+        [InlineData("12345", "12345", 4, "123")]
+        [InlineData("12345", "12345", 0, "")]
+        [InlineData("12345", "12345", 11, "1234512345")]
+        [InlineData("", "", 0, "")]
```

Comment:
we need a test for a large buffer, to ensure the zero fill works as expected

Like `[InlineData("12345", "12345", 32, "1234512345")]` and verify the zero padding at the end (also initialize all of `dst` with something != 0)